packages/common/package.json

```diff
@@ -53,6 +53,7 @@
   "devDependencies": {
     "@vitest/browser": "^3.2.4",
     "jsdom": "^26.1.0",
+    "npm-run-all2": "^8.0.4",
```

Comment:
One strange change is now if you want to run scripts (like `npm-run-all`) the package must be installed in the workspace - it doesn't automatically go run the top level. So I've added a few here to get ci to pass...